test/unit_test/cb_las_spanner_test.cc

```diff
@@ -601,26 +601,36 @@ BOOST_AUTO_TEST_CASE(check_spanner_rejects_same_actions)
 
 BOOST_AUTO_TEST_CASE(check_spanner_rejects_actions_that_are_linear_combinations_of_other_actions)
```

Comment:
Test name seems inconsistent with result. Is the expected behavior "reject" or "accept" the linear combination action?

Reply:
made it more generic